src/Raven.Embedded/ServerOptions.cs

```diff
@@ -92,7 +94,18 @@ public class SecurityOptions
             public string CertificateLoadExecArguments { get; internal set; }
             public string ServerCertificateThumbprint { get; internal set; }
         }
-    }
 
+        public LicenseOptions LicenseConfiguration { get; set; } = new();
 
+        public class LicenseOptions
+        {
+            public string License { get; set; }
+            public string LicensePath { get; set; }
+            public bool EulaAccepted { get; set; }
+            public bool DisableAutoUpdate { get; set; }
+            public bool DisableAutoUpdateFromApi { get; set; }
+            public bool DisableLicenseSupportCheck { get; set; }
+            public bool EnforceLicense { get; set; }
```

Comment:
maybe ThrowOnLicenseActivationFailure ?

Reply:
I was thinking that it might not be clear enough that an exception will also be thrown if there's no license at all. No license means nothing to activate, so it might seem like there wouldn't be an activation failure and no exception.

How about we go with `ThrowOnInvalidOrMissingLicense` instead? This way, it's clear that an exception will be thrown if the license is invalid or missing.

Reply:
works for me

Reply:
Done.